readme.md

```diff
@@ -136,6 +136,7 @@
 - [Images](https://github.com/heyalexej/awesome-images)
 - [Creative Commons Media](https://github.com/shime/creative-commons-media)
 - [Database](https://github.com/numetriclabz/awesome-db)
+- [Mysql](http://shlomi-noach.github.io/awesome-mysql) 
```

Comment:
Wrong capitalization and the link should be to the repo, so https://github.com/shlomi-noach/awesome-mysql/blob/gh-pages/index.md